.github/workflows/build_playground_backend.yml

```diff
@@ -81,21 +81,30 @@ jobs:
       - name: Preapre Build Go
         run: ./gradlew playground:backend:containers:go:dockerTagPush -Pdocker-repository-root='${{ secrets.PLAYGROUND_REGISTRY_NAME}}/${{ secrets.PLAYGROUND_PROJECT_ID }}/playground-repository' -Pdocker-tag=${{ env.DOCKERTAG }}
         if: startsWith(github.ref, 'refs/tags/')
+      - name: Preapre Build Router
```

Comment:
@andreykus (cc: @daria-malkova) Please correct spelling of 'Prepare' in a future PR.